pandas/tests/dtypes/test_dtypes.py

```diff
@@ -323,6 +324,13 @@ def test_tz_standardize(self):
         dtype = DatetimeTZDtype("ns", dr[0].tz)
         assert dtype.tz == tz
 
+    def test_dfgroupby_shift_tz(self):
+        # GH 30134
+        df = pd.DataFrame({"a": [1, 1], "date": dt.datetime.now(tz=pytz.utc)})
```

Comment:
```suggestion
        df = pd.DataFrame({"a": [1, 1], "date": pd.Timestamp.utcnow()})
```

Slightly more idiomatic